site/docs/v1/tech/reference/cookies.adoc

```diff
@@ -0,0 +1,108 @@
+---
+layout: doc
+title: Hosted Login Pages Cookies
+description: A list of cookies set by the FusionAuth Hosted Login pages
+navcategory: admin
+---
+
+== Overview
+
+Cookies are a critical part of web applications, and FusionAuth sets cookies. When you call certain APIs, such as the Login API, cookies may be set. These cookies are specified in the link:/docs/v1/tech/apis[API documentation].
+
+When you use the link:/docs/v1/tech/core-concepts/integration-points#hosted-login-pages[hosted login pages], FusionAuth sets cookies to enable functionality.
+
+== Domains
+
+The domain of all cookies is limited to the domain on which the FusionAuth instance is running. You can control the domain FusionAuth uses by link:/docs/v1/tech/admin-guide/proxy-setup[setting up a proxy].
+
+In other words, if FusionAuth lives at `auth.piedpiper.com`, it will only set cookies for that domain. It will never set cookies for `.piedpiper.com`.
+
+== Cookie List
+
+[WARNING.warning]
+====
+Cookies set by the hosted login pages are displayed here for informational purposes only.
+
+Duration, name, and purpose are subject to change.
+====
+
+When a cookie has a value of `Yes` for the `Prefix` column, name is used as a prefix for one or more cookies.
+
+//Pulled from Cookies.java
+
+.Hosted Login Page Cookies 
+[cols="2,1,1,3"]
+|===
+| Name | Prefix | Duration | Function
```

Comment:
I would suggest Name, HTTP Only, and Function? Do we care about duration and prefix?

Reply:
Duration and domain were in the client request that led to this doc.

Prefix is useful because it indicates there are multiple cookies of a certain type. I get that it is awkward, let me see if I can think of a better way to convey it.